packages/wonder-blocks-modal/components/standard-modal.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+import * as React from "react";
+import {StyleSheet, css} from "aphrodite";
+
+import Color from "wonder-blocks-color";
+import {View} from "wonder-blocks-core";
+import {HeadingMedium} from "wonder-blocks-typography";
+
+type Props = {
+    /**
+     * The content of the modal, appearing between the header and footer.
+     */
+    content: React.Node,
+
+    /**
+     * The title of the modal, appearing in the header.
+     *
+     * If a subtitle is also present, this becomes smaller to accommodate both
+     * within the title bar.
+     */
+    title: string,
+
+    /**
+     * The subtitle of the modal, appearing in the header, below the title.
+     */
+    subtitle?: string,
+
+    /**
+     * The content of the modal's footer. A great place for buttons!
+     *
+     * Content is right-aligned by default. To control alignment yourself,
+     * provide a container element with 100% width.
+     */
+    footer?: React.Node,
+};
```

Comment:
Needs a boolean `showHeader?: boolean`.  I need to update the spec since I thought we had to handle multiple colors.

Reply:
Like, to determine whether the header is dark or light?

Reply:
Oh, got it, header vs title bar, nvm.

Reply:
In that case, do we need something like `header?: React.Node`, in order to satisfy cases like the "Subject picker" example where the header has unique content?

Reply:
I was told that the we aren't doing things like that anymore.  The tricky thing is that the new designs call for the header to collapse when scrolling up but this is not what happens currently with the subject picker.  It's also non-obvious how to collapse a header image into a titlebar background.

Reply:
We talked about this in slack and will defer work on the header b/c there are still unanswered questions.